BedrockServer.cpp

```diff
@@ -206,6 +206,12 @@ void BedrockServer::sync(SData& args,
         replicationState.store(nodeState);
         masterVersion.store(syncNode.getMasterVersion());
 
+        // If we're not mastering, we turn off multi-write until we've finished upgrading the DB. This persists until
```

Comment:
I'll wait for @quinthar to chime in on this, but I wonder if there isn't a better more global way to turn off multi-write when not mastering or something. Just wondering what other edge cases are out there.